src/services/iiif-parser.js

```diff
@@ -96,6 +98,10 @@ export function getMediaInfo(manifest, canvasIndex) {
   }
 }
 
+export function getTracks() {
+  return getReduxManifest().getSeeAlso();
```

Comment:
We will probably need to refine this to filter the see also, but this works for now.